docs/tutorials/azure-private-dns.md

```diff
@@ -101,26 +60,24 @@ $ az network private-dns link vnet create -g externaldns -n mylink \
 ```
 
 ## Configure service principal for managing the zone
-ExternalDNS needs permissions to make changes in Azure Private DNS.  
+ExternalDNS needs permissions to make changes in Azure Private DNS.
 These permissions are roles assigned to the service principal used by ExternalDNS.
 
-A service principal with a minimum access level of `Private DNS Zone Contributor` to the Private DNS zone(s) and `Reader` to the resource group containing the Azure Private DNS zone(s) is necessary.
-More powerful role-assignments like `Owner` or assignments on subscription-level work too. 
+A service principal with a minimum access level of `Private DNS Zone
+Contributor` to the Private DNS zone(s) and `Reader` to the resource group
+containing the Azure Private DNS zone(s) is necessary.  More powerful
+role-assignments like `Owner` or assignments on subscription-level work too.
 
-Start off by **creating the service principal** without role-assignments.
-```
-$ az ad sp create-for-rbac --skip-assignment -n http://externaldns-sp
-{
-  "appId": "appId GUID",  <-- aadClientId value
-  ...
-  "password": "password",  <-- aadClientSecret value
-  "tenant": "AzureAD Tenant Id"  <-- tenantId value
-}
-```
-> Note: Alternatively, you can issue `az account show --query "tenantId"` to retrieve the id of your AAD Tenant too.
+Start off by **creating the service principal** without role-assignments.  ```
+$ az ad sp create-for-rbac --skip-assignment -n http://externaldns-sp {
+"appId": "appId GUID",  <-- aadClientId value ...  "password": "password",  <--
+aadClientSecret value "tenant": "AzureAD Tenant Id"  <-- tenantId value } ```
```

Comment:
Why wrap this? It was much more readable before.

Reply:
I think this is my text editor that did an unintended change formatting the file, I will revert this